solidity/random-beacon/hardhat.config.ts

```diff
@@ -94,8 +94,8 @@ const config: HardhatUserConfig = {
     ropsten: {
       url: process.env.CHAIN_API_URL || "",
       chainId: 3,
-      accounts: process.env.CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY
-        ? [process.env.CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY]
+      accounts: process.env.ACCOUNTS_PRIVATE_KEYS
+        ? process.env.ACCOUNTS_PRIVATE_KEYS.split(",")
```

Comment:
If we change the name of this property, we should also update it here:
https://github.com/keep-network/keep-core/blob/144e7698646e5d26e21efaadf79d5a564635ceb7/.github/workflows/contracts-random-beacon.yml#L187

Reply:
Also, don't we need to make similar change to the `solidity/ecdsa/hardhat.config.ts`? Currently in the CI we pass the value of `CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY` in both projects by filling it with the same GH secret. If we were to provide comma-separated accounts in that secret, with your changes it would work only for the `random-beacon` deploy job.

Reply:
cfefd399429d6d23d9f872d0a0034be295b21fdf